installer/datafiles/Base_SCXCore.data

```diff
@@ -258,6 +272,9 @@ set -e
 %Postinstall_200
 CreateSoftLinkToSudo
 
+%Postinstall_250
+CreateSoftLinkToTmpDir
```

Comment:
No need to create a new section here.  Just do something like:

%Postinstall_200
CreateSoftLinkToSudo
CreateSoftLinkToTmpDir